niworkflows/interfaces/utils.py

```diff
@@ -589,10 +589,8 @@ class AddTSVHeader(SimpleInterface):
 
     .. testsetup::
 
-    >>> import pandas as pd
-    >>> from tempfile import TemporaryDirectory
-    >>> tmpdir = TemporaryDirectory()
-    >>> os.chdir(tmpdir.name)
+    >>> cwd = os.getcwd()
+    >>> os.chdir(tmpdir)
```

Comment:
If you don't mind `Path(tmpdir / 'data.tsv')` below, we could even drop the `chdir` altogether.